pkg/piecestore/psserver/psdb/testdata/sqlite.v0.sql

```diff
@@ -5,6 +5,5 @@ CREATE INDEX idx_ttl_expires ON ttl (expires);
 
 -- NEW DATA --
 
-INSERT INTO ttl VALUES(1,2,3,4);
```

Comment:
you shoudn't remove this. This value should be in DB before your migration. It shouldn't be in DB after your migration.

Reply:
done